crates/editor/src/element.rs

```diff
@@ -2680,15 +2679,10 @@ impl EditorElement {
         let cursor_ranges = layout
             .cursors
             .iter()
-            .map(|cursor| {
-                let point = cursor
-                    .0
-                    .to_display_point(&layout.position_map.snapshot.display_snapshot);
-                ColoredRange {
-                    start: point.row(),
-                    end: point.row(),
-                    color: cursor.1,
-                }
+            .map(|cursor| ColoredRange {
+                start: cursor.0.row(),
```

Comment:
NIT: could destructure `cursor` for readability.

Reply:
Destructured